src/ArcxAnalyticsSdk.ts

```diff
@@ -158,3 +188,7 @@ type FirstVisitPageType = {
     campaign: string | null
   }
 }
+
+function getWeb3Provider(): MetaMaskInpageProvider | undefined {
```

Comment:
There's no need to make this function public imo

Reply:
this function is not public